desktop-widgets/tab-widgets/maintab.cpp

```diff
@@ -204,8 +204,6 @@ MainTab::MainTab(QWidget *parent) : QTabWidget(parent),
 	// enable URL clickability in notes:
 	new TextHyperlinkEventFilter(ui.notes);//destroyed when ui.notes is destroyed
 
-	acceptingEdit = false;
```

Comment:
this causes me flashbacks... I think we added this because there was some way that we ended up recursing into our own code and without this flag we did rather weird stuff. I am not sure that this is still true and I am definitely not at all sure that your code is wrong... I just remember that this was a major source of pain early on in the Qt code.

Reply:
Note that this commit does *not* remove the `acceptingEdit` functionality. Rather it unifies it with the `editMode == IGNORE` functionality that does exactly the same thing.